idea_town/frontend/static-src/app/views/experiment-page.js

```diff
@@ -42,13 +42,16 @@ export default PageView.extend({
       hook: 'feedback'
     }],
 
-    'model.modified_date': {
-      type: (el, value) => new Date(value),
+    'model.modified': {
+      type: (el, value) => {
+        const d = new Date(value);
+        el.innerHTML = `${d.getMonth() + 1} / ${d.getDate()} / ${d.getFullYear()}`;
```

Comment:
Actually, on that note, not sure if you want to verify `d` before trying to use `.getMonth()` and gang.
Otherwise we may get something back like "NaN/NaN/NaN" or "Invalid Date" if you pass in an empty string or unexpected value.